functions/Get-DbaCollations.ps1

```diff
@@ -0,0 +1,104 @@
+function Get-DbaCollations {
+	<#
+		.SYNOPSIS
+			Function to get available collations for a given SQL Server
+
+		.DESCRIPTION
+			The Get-DbaCollations function returns the list of collations available on each SQL Server.
+			Only the connect permission is required to get this information.
+
+		.PARAMETER SqlInstance
+			The SQL Server instance, or instances. Only connect permission is required.
+
+		.PARAMETER SqlCredential
+			SqlCredential object to connect as. If not specified, current Windows login will be used.
+
+		.PARAMETER Detailed
+			Lookup Locale and Culture names (slightly slower)
+
+		.PARAMETER Silent
+			Use this switch to disable any kind of verbose messages
+
+		.NOTES
+			Author: Bryan Hamby (@galador)
+
+			Website: https://dbatools.io
+			Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+			License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+		.LINK
+			https://dbatools.io/Get-DbaCollations
+
+		.EXAMPLE
+			Get-DbaCollations -SqlInstance sql2016
+
+			Gets all the collations from server sql2016 using NT authentication 
+
+	#>
+	[CmdletBinding()]
+	Param (
+		[parameter(Position = 0, Mandatory = $true, ValueFromPipeline = $true)]
+		[Alias("ServerInstance", "SqlServer")]
+		[DbaInstanceParameter[]]$SqlInstance,
+		[PSCredential]$SqlCredential,
+		[switch]$Detailed,
+		[switch]$Silent
+	)
+	
+	process {
+		function Get-CodePageDescription ($CodePage) {
+			$encoding = [System.Text.Encoding]::GetEncoding($CodePage)
```

Comment:
Unless I'm missing something, doesn't `Get-Language` do the exact same thing I'm doing? I could (and probably will, no use in duplication) change my function to use `Get-Language`, but wouldn't that still just return the cultures that are on the local machine?